tests/coalaTest.py

```diff
@@ -42,44 +38,23 @@ def test_did_nothing(self):
 
     def test_show_all_bears(self):
         with bear_test_module():
-            retval, output = execute_coala(coala.main, "coala", "-A")
+            retval, output = execute_coala(coala.main, "coala", "-B")
             self.assertEqual(retval, 0)
-
-            lines = output.splitlines()
-            bear_lines = sum(1 for line in lines if line.startswith("==="))
-            self.assertEqual(bear_lines, 4)
-
-            for line in lines:
-                self.assertNotIn("WARNING", line)
-
-    def test_show_bears(self):
-        with bear_test_module():
-            retval, output = execute_coala(
-                coala.main, "coala", "-B",
-                "-b", "LineCountTestBear, SpaceConsistencyTestBear",
-                "-c", os.devnull)
-            self.assertEqual(retval, 0)
-            self.assertIn(LineCountTestBear.run.__doc__.strip(), output)
+            self.assertEqual(len(output.splitlines()), 4)
 
     def test_show_language_bears(self):
         with bear_test_module():
-            retval, output = execute_coala(coala.main, "coala", "-l", "java")
+            retval, output = execute_coala(
+                coala.main, "coala", "-B", "-l", "java")
             self.assertEqual(retval, 0)
-            lines = output.splitlines()
-            bear_missing_lines = sum(1 for line in lines if "WARNING" in line)
-            self.assertEqual(bear_missing_lines, 0)
-
-            self.assertIn(JavaTestBear.run.__doc__.strip(), output)
-            self.assertIn(LineCountTestBear.run.__doc__.strip(), output)
-            self.assertNotIn(SpaceConsistencyTestBear.run.__doc__.strip(),
-                             output)
+            self.assertEqual(len(output.splitlines()), 2)
```

Comment:
this is only for the test bears, not for the installed ones. We only have four test bears and two of them are for java, I think this is a legit test.